ethereum/core/src/main/java/org/hyperledger/besu/ethereum/privacy/PrivateTransaction.java

```diff
@@ -565,7 +565,7 @@ private static Bytes32 computeSenderRecoveryHash(
         RLP.encode(
             out -> {
               out.startList();
-              out.writeLongScalar(nonce);
+              out.writeUnsignedLongScalar(nonce);
```

Comment:
Also L465 in this file
    out.writeLongScalar(t.getNonce());

Reply:
Thank you! I've changed them now and checked again, I think now all occurences for the nonce have been changed